pkg/auth/authmap_gc.go

```diff
@@ -39,7 +39,10 @@ type authMapGarbageCollector struct {
 	ciliumIdentitiesSynced     bool
 	ciliumIdentitiesDeleted    map[identity.NumericIdentity]struct{}
 
-	endpointRepo endpointRepository
+	endpointRepo         endpointRepository
```

Comment:
`endpointRepo` is no longer used -> delete field

Reply:
good catch!